src/workerd/server/BUILD.bazel

```diff
@@ -123,7 +123,6 @@ wd_cc_library(
         "//src/workerd/io",
         "//src/workerd/io:worker-entrypoint",
         "//src/workerd/jsg",
-        "//src/workerd/util:perfetto",
```

Comment:
Nit: Same as above – we should follow IWYU here so that we don't depend on transitively pulling in util:perfetto through some other target here.

Reply:
Similarly, we should add an explicit dependency on `//src/workerd/api/node` here.